openmdao/drivers/genetic_algorithm_driver.py

```diff
@@ -437,8 +448,9 @@ def execute_ga(self, vlb, vub, bits, pop_size, max_gen, random_state):
         self.npop = int(pop_size)
         fitness = np.zeros((self.npop, ))
 
-        Pc = 0.5
-        Pm = (self.lchrom + 1.0) / (2.0 * pop_size * np.sum(bits))
+        # If mutation rate is not provided as input
+        if Pm is None:
+            Pm = (self.lchrom + 1.0) / (2.0 * pop_size * np.sum(bits))
         elite = self.elite
 
         # TODO: from an user-supplied intial population
```

Comment:
I would probably just add a small test where you set these (just to verify that options in the api can be set.) Probably doesn't need to be a big test.

Reply:
Test added.